<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedPainterClass</class>
 <widget class="QMainWindow" name="SharedPainterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SharedPainter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="painterView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnectDirect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>Server</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionScreen_Shot">
   <property name="text">
    <string>Screen Shot</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionBroadCastServerType">
   <property name="text">
    <string>Server</string>
   </property>
  </action>
  <action name="actionBroadCastClientType">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sharedpainter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>SharedPainterClass</receiver>
   <slot>actionExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedConnectButton()</slot>
  <slot>clickedUndoButton()</slot>
  <slot>clickedStartServerButton()</slot>
  <slot>clickedStartClientButton()</slot>
  <slot>clickedScreenShotButton()</slot>
  <slot>actionExit()</slot>
 </slots>
</ui>
